lab-justin/test/storage-unit-test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Storage = require('../lib/storage');
+const testStoreage = new Storage(`${__dirname}/data`);
+
+describe('testing storage module', function(){
+  describe('testing setItem', function(){
+    it('should return the item', function(done){
+      testStoreage.setItem('note', {id: 123456, content: 'test test test'})
```

Comment:
You should have your gitignore ignore your note folder and everything inside of it, otherwise when I clone it down I don't know if this test passes because it works or because the directory it's supposed to write to and the content that's supposed to be inside of it are already there.